README.md

```diff
@@ -1,2 +1,131 @@
 # node-setup
+
 Connected Cars JavaScript/TypeScript development setup
+
+## Use
+
+``` bash
+npm install https://github.com/connectedcars/node-setup
+```
+
+### Automatic
+
+Don't overwrite existing files:
+
+``` bash
+./node_modules/.bin/setup init
```

Comment:
Wouldn't we want to run this automatically (on a postinstall) when installing this package?

Reply:
Right now it's manual as I think it needs a bit more work, fx. support merging of the config templates so we don't overwrite customizations in the project, but it could be the long term goal.